server/helpers/file_helpers.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+import boto3
+from flask import current_app as app
+from PIL.JpegImagePlugin import JpegImageFile
+from werkzeug.datastructures import FileStorage
+
+from server.helpers.app_helpers import is_production
+
+
+def save_image_file_locally(image: JpegImageFile, image_name: str) -> None:
+
+    image_directory = app.config["IMAGE_DIRECTORY"]
```

Comment:
app.config.get('IMAGE_DIRECTORY', '.')

Reply:
This is resolved if the user setups up correctly using the makefile: https://github.com/Kayra/metuo/blob/master/Makefile#L23
Explicit better than implicit and all that ;)